Makefile

```diff
@@ -92,8 +92,11 @@ generate-protoc: require-protoc require-protoc-gen-gofast
 generate-stringer:
 	go generate github.com/pilosa/pilosa
 
+generate-pql: require-peg
+	cd pql && peg -inline -switch pql.peg && cd ..
```

Comment:
the peg readme says to use `-switch` "at your own peril", what are the drawbacks/risks?